jobs/integr8ly/ocp4/ocp4-testing-executor.yaml

```diff
@@ -114,12 +131,15 @@
           name: TO_DO
           description: "It specifies what stages of the pipeline will be executed. Provision and deprovision are for non-pds clusters only"
           choices:
-            - provision + install + deprovision
+            - provision + install + tests + deprovision
+            - provision + install + tests
             - provision + install
+            - install + tests
             - install
     dsl: |
+        #!groovy
```

Comment:
This is not required. Is it?

Reply:
I am not sure. Probably the same story as with `#!/bin/bash` in shell scripts. Don't have any preference though

Reply:
fair enough. I will remove it, since it might be redundant